draft-looker-jwt-cwt-status-list.md

```diff
@@ -32,38 +32,57 @@ This specification defines a status list representation and processing rules for
 
 # Introduction
 
-JSON Web Tokens (JWTs) {{RFC7519}} and CBOR Web Tokens (CWTs) {{RFC8392}} as secure token formats, have vast possible applications. Some of these applications can involve issuing a token whereby certain semantics about the token can change over time which are important to be able to communicate to relying parties in an interoperable manner, such as whether the token is considered revoked by its issuer.
+JSON Web Tokens (JWTs) {{RFC7519}} and CBOR Web Tokens (CWTs) {{RFC8392}} as secure token formats, have vast possible applications. Some of these applications can involve issuing a token whereby certain semantics about the token can change over time which are important to be able to communicate to relying parties in an interoperable manner, such as whether the token is considered suspended or revoked by its issuer.
 
-This document defines a status list using JWT and CWT for representation that is capable of communicating the individual statuses of multiple tokens. The document also defines how an issuer of a token references a status list in a JWT or CWT which has a status to convey.
+This document defines a Status List in JWT and CWT representation that describes the individual statuses of multiple Subject Tokens. The Status List contains binary data, where each Subject Token corresponds with a certain position within a bit array(also known as bit vector, string or list). The document also defines how an issuer of a Subject Token in JWT or CTW representation references a Status List to convey status data like suspension or revocation of the Subject Token and related information. Status Lists may be composed for a range of various Status Types with one or more bits, the document defines two configurations for the most common use cases as well as an extensibility mechanism for custom Status List Types. The Status List may be used by a Status List Provider in the Issuer-Holder-Verifier model, as described in {{RFC-XXXX}} to express the status of verifiable credentials (Subject Tokens) issued by an issuer.
```

Comment:
```suggestion
This document defines a status list in JWT and CWT representation that describes the individual statuses of multiple subject tokens. The statuses of each subject token are conveyed via a bit array in the status list. Each subject token during issuance is allocated an index which represents a position within this bit array and the value of the bit(s) at this position correspond to the subject tokens status. The document also defines how an issuer of a subject token in JWT or CTW representation references a status list. Status lists may be composed for expressing a range of status types, the document defines two configurations for the most common use cases as well as an extensibility mechanism for custom status list types. The status list may be used by an issuer in the Issuer-Holder-Verifier model, as described in (XXX) to express the status of verifiable credentials (Subject Tokens) issued by an issuer.
```

Reply:
Bit of a suggested revision to the text here.

Reply:
@tplooker Looking at other RFCs like JWT, I see that things that are mentioned in the terminology are capitalized throughout the text. Why does this not apply here?